ocaml/runtime/shared_heap.c

```diff
@@ -70,6 +70,14 @@ static struct {
   caml_plat_mutex lock;
   pool* free;
 
+  /* Mapped but not yet active pools */
+  uintnat fresh_pools;
+  char* next_fresh_pool;
+
+  /* Counts pools in some domain's heap, or in the global list below.
```

Comment:
It took me a little while to understand this comment. I think something like "Count of all pools that are in-use across all domains and the global free list"